release_docs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+set -x
+if [ "${TRAVIS_REPO_SLUG}" = "" ];
+then
+  echo "Cannot release docs without TRAVIS_REPO_SLUG set"
+  exit 0;
+fi
+REPO="https://github.com/${TRAVIS_REPO_SLUG}"
+
+rm -rf docs
+git clone -b gh-pages --single-branch $REPO ./docs
+cd docs
+git pull origin gh-pages
+cd ..
+
+DEST="master"
+
+if [ "${TRAVIS_TAG}" != "" ];
+then
+  DEST="${TRAVIS_TAG}"
+  # change the default page to the latest
+  echo "<meta http-equiv='refresh' content='0; url=/Parse-SDK-JS/api/${DEST}'>" > "docs/api/index.html"
```

Comment:
This is interesting... So per release we're just redirecting via meta to the latest set? That's kind of cool actually. One thing I'm curious about is if we're maintaining an index or a table of contents for existing doc versions?

Reply:
So far we don’t, but with this strategy we’ll be able to :)